### DISCUSSION TITLE
future3 how to?

### DISCUSSION THREAD

**Discussion by passuff** [MiczFlor/RPi-Jukebox-RFID#1844] [Q&A]
I successfully installed the future3 release for the first time and now I'm wondering where to start. Where to put the files? Which files can be read? How to make playlists (with sorting)? .... many more questions.. I read the documentation but its mainly focusing on installation but not on usage..
Any idea where to start?

**Comment by joka80** [comment]
I'm asking me that as well. I have invested some time on searching online and in documentations. Had older versions installed and everything was self-explanatory. 
But with version future3, either I think in a too complex way or phoniebox just got less intuitive.

Could anyone pls tell me, where I can upload and manage new audio files?

Thanks a lot in advance

**Comment by s-martin** [comment]
Right now there’s https://rpi-jukebox-rfid.readthedocs.io/en/latest - but it’s not yet a comprehensive user guide.

**Comment by alesrosina** [comment]
What's the status of this version? Docs seems to be gone now ....

**Reply by AlvinSchiller** [reply]
Docs have been moved to the wiki on `future3/develop` -> https://github.com/MiczFlor/RPi-Jukebox-RFID/tree/future3/develop/documentation 
There are currently ongoing changes e.g. to support Bookworm. A new Release should be available soon.

**Reply by pabera** [reply]
I realize we have to make this more visible.  2 things need to be done

1. Organize the Docs better so they work as a website in GitHub
2. Merge future3/develop into future3/main to be more accessible
3. Update main readme to lead to the correct docs. We should also update the state of future3 there

**Reply by alesrosina** [reply]
Yes, i was searching for it, but could found only few discussions mentioninf this release.

So that would be very much appreciated.

What I'm also wondering is - is the new version already solid as previous version? 

Is there any performance gain? I'm running it on raspberry pi 1, which is quite old, but it does the job. Only startup time is really slow, it takes about 5min to get to startup sound on Classic version.
